themes/nekonight/nekonight-base.sh

```diff
@@ -0,0 +1,35 @@
+function _omb_theme_nekonight_git_prompt_info() {
+  local branch_name
+  branch_name=$(git symbolic-ref --short HEAD 2>&-)
```

Comment:
I also forgot to check it, but we also want to call `git` through `_omb_prompt_git` to avoid hitting user's alias `git` (and also to introduce timeouts #173 and protection against `core.fsmonitor` #428 in the future).

```suggestion
  branch_name=$(_omb_prompt_git symbolic-ref --short HEAD 2>&-)
```

All the other calls of `git` should also use `_omb_prompt_git`.

Reply:
> All the other calls of `git` should also use `_omb_prompt_git`.

@BrunoCiccarino

Reply:
There are still several calls of `git`.

Reply:
> @BrunoCiccarino
> 
> > All the other calls of `git` should also use `_omb_prompt_git`.

I'm going to change everything there, I was studying bash today that's why I disappeared

Reply:
Thanks. Please let me know when you complete it.

Reply:
Thanks! I've confirmed.